tests/unittests/tasks/test_channels.py

```diff
@@ -0,0 +1,502 @@
+from __future__ import annotations
+
+import json
+
+import pytest
+
+from scenario_player.exceptions import (
+    RESTAPIStatusMismatchError,
+    ScenarioAssertionError,
+    ScenarioError,
+)
+from scenario_player.tasks.channels import STORAGE_KEY_CHANNEL_INFO
+
+# TODO: Add tests for request timeouts
+
+
+@pytest.mark.parametrize(
+    (
+        "task_name",
+        "task_params",
+        "expected_exception",
+        "expected_exception_message",
+        "expected_req_method",
+        "expected_req_url",
+        "expected_req_body",
+        "resp_code",
+        "resp_json",
+    ),
```

Comment:
A general pointer for consistency: I state `argnames` as a string in tests, and pass `argvalues` as a keyword argument. I think it helps reading the parametrize mark.

Reply:
I'm not a fan of mangling the argnames into a single string. That's why we have tuples ;)

Also I'd argue we should be using the same code style here as everywhere else in raiden.

Reply:
> Also I'd argue we should be using the same code style here as everywhere else in raiden.

Yea, that's a pretty good argument.